Documentation/MigrationGuide.md

```diff
@@ -5,52 +5,55 @@ The two general themes for this version are:
 - Templates now reside in the subfolder corresponding to their subcommand (`colors`, `fonts`, ...) intead of the filename being prefixed with it.
 - The `default` template doesn't exist anymore, templates now specify which swift version they support.
 
-Below is a list of renamed and removed templates, grouped by subcommand. If your template isn't listed, you don't need to do anything.
+Below is a list of renamed and removed templates, grouped by subcommand. If your template hasn't been renamed or removed, you don't need to do anything. You still might want to review the documentation for that template, as there might be new features you may be interested in.
 
 ### Colors
 
 | Old | New | Reason |
 | --- | --- | ------ |
-| default | swift2 | |
-| rawvalues | **deleted** | Seldomly used |
+| `default` | ➡️ `swift2` | |
+| `rawvalues` | ❌ _deleted_ | Seldomly used |
+| `swift3` | ✅ `swift3` | |
 
 ### Fonts
 
 | Old | New | Reason |
 | --- | --- | ------ |
-| default | swift2 | |
+| `default` | ➡️ `swift2` | |
+| `swift3` | ✅ `swift3` | |
 
 ### Images / XCAssets
 
 | Old | New | Reason |
 | --- | --- | ------ |
-| all-values | **deleted** | The other templates by default now generate an `allValues` static constant |
-| default | **deleted** | Deprecated by `dot-syntax` (now called `swift2`) |
-| dot-syntax | swift2 | |
-| dot-syntax-swift3 | swift3 | |
-| swift3 | **deleted** | Deprecated by `dot-syntax-swift3` (now called `swift3`) |
+| `all-values` | ❌ _deleted_ | The other templates by default now generate an `allValues` static constant |
+| `default` | ❌ _deleted_ | Deprecated by `dot-syntax` (now called `swift2`) |
+| `dot-syntax` | ➡️ `swift2` | |
+| `dot-syntax-swift3` | ➡️ `swift3` | |
+| `swift3` | ❌ _deleted_ | Deprecated by `dot-syntax-swift3` (now called `swift3`) |
```

Comment:
Mmmh that's a tough case this one. In a sense, It has been "renamed to the same name" 😄
Maybe use a warning emoji to tell there's no name change required from the user but they have to be aware of a change in the output?

```
| `swift3` | ⚠️ swift3 | Now integrates the recursive features of the previously named `dot-syntax-swift3` template |
```